src/style.css

```diff
@@ -0,0 +1,76 @@
+.container {
+  height: 160px;
+  width: 140px;
+}
+
+.star--0 {
+  display: flex;
+  background-image: url(./images/star.svg),
+    url(./images/star.svg),
+    url(./images/star.svg),
+    url(./images/star.svg),
+    url(./images/star.svg);
+  background-position: 0.01px, 16px, 32px, 48px, 64px;
+  background-repeat: no-repeat;
+  height: 14px;
+}
+
```

Comment:
To style all `stars__star` `div`s you can use` :nth-child` pseudo-class

Example: 

```
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n+2),
.... so on
{
  background-image: url(images/star-active.svg);
}
```